portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -477,6 +477,28 @@ definition {
 			expected = "${editStructuredContentId}");
 	}
 
+	macro assertStructuredContentTitleIsExpectedWithCorrectOrder {
+		Variables.assertDefined(parameterList = "${responseToParse}");
+
+		var structuredContentIdActual = JSONUtil.getWithJSONPath("${responseToParse}", "$..title");
+
+		var structuredContentIdActual = ListUtil.newListFromString("${structuredContentIdActual}");
+
+		var size = ListUtil.size("${structuredContentIdActual}");
+		var index = "0";
+
+		while ("${index}" != "${size}") {
+			var structuredContentIdexpected = JSONUtil.getWithJSONPath("${responseToParse}", "$..['items'][${index}].title");
+			var locale = ListUtil.get("${structuredContentIdActual}", "${index}");
+
+			TestUtils.assertEquals(
+				actual = "${locale}",
+				expected = "${structuredContentIdexpected}");
```

Comment:
I think now we have a check if the first title also appears as first in the answer.